src/kdbindings/signal.h

```diff
@@ -207,6 +41,14 @@ class ConnectionHandle
  *
  * The Args type parameter pack describe which value types the Signal will emit.
  *
+ * Deferred Connection:
```

Comment:
I'd mark it with `@warning` and tell that it's experimental still. We should probably first try validating it in KDFoundation and then remove the warning. What do you think @LeonMatthesKDAB @phyBrackets ?

Reply:
Sounds like a good idea.
Done.